File: CPP/beginner/1009.cpp
#include <stdio.h>
 
int main() {
    
    char Name[50];
    double Salary = 0, Products = 0, Bonus = 0, Total = 0;
    
    scanf("%s", &Name);
    scanf("%lf", &Salary);
    scanf("%lf", &Products);
    
    Bonus = (15 * Products) / 100;
    
    Total = Salary + Bonus;
    
    printf("TOTAL = R$ %.2lf\n", Total);
   
    return 0;
}

File: CSharp/beginner/1009.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string name = Console.ReadLine();
        double salary = double.Parse(Console.ReadLine());
        double sales = double.Parse(Console.ReadLine());
        
        double bonus = (sales / 100) * 15;
        double total = salary + bonus;

        Console.WriteLine($"TOTAL = R$ {total:F2}");
    }
}
